source/catalog.html

```diff
@@ -1,239 +1,290 @@
 <!DOCTYPE html>
-<html lang="ru">
+<html class="page" lang="ru">
 
 <head>
   <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="stylesheet" href="css/style.css">
   <title>Каталог</title>
 </head>
 
-<body>
+<body class="page__body">
   <header class="header">
+    <a class="header__logo" href="#">
+      <img src="img/images/logo-1.svg" width="70" height="59" alt="логотип Cat Energy">
+    </a>
+    <a class="header__logo" href="#">
+      <img src="img/images/logo-2.svg" width="101" height="18" alt="cat energy">
+    </a>
+    <button class="nav__button-header" type="button"><span class="visually-hidden">Открыть меню</span></button>
     <nav class="header__nav nav">
-      <a class="nav__logo" href="#">
-        <img src="img/images/logo-1.svg" width="70" height="59" alt="логотип Cat Energy">
-        <img src="img/images/logo-2.svg" width="101" height="18" alt="cat energy">
-      </a>
-      <button class="header__button-header" type="button"><span class="visually-hidden">Открыть меню</span></button>
+      <ul class="nav__list">
+        <li class="nav__item">главная</li>
+        <li class="nav__item">каталог продукции</li>
+        <li class="nav__item">подбор программы</li>
+      </ul>
```

Comment:
Это ж ссылки. На другие страницы прописываешь ссылки. На текущую - нет. В остальных страницах так же.

```suggestion
      <ul class="nav__list">
        <li class="nav__item"><a href="index.html">главная</a></li>
        <li class="nav__item"><a>каталог продукции</a></li>
        <li class="nav__item"><a href="form.html">подбор программы</a></li>
      </ul>
```